app/helpers/report_helper/report_information_helper.rb

```diff
@@ -44,7 +44,7 @@ def report_schedule(schedules)
       rows = []
       schedules.each do |s|
         tz = s.run_at && s.run_at[:tz] ? s.run_at[:tz] : session[:user_tz]
-        schedule_url = role_allows?(:feature => 'miq_report_schedules') ? report_action_url("msc-#{s.id}", "schedules") : ""
+        schedule_url = role_allows?(:feature => 'miq_report_schedules') ? report_action_url("msc-#{s.id}", "schedules") : {}
```

Comment:
namimg-wise it's weird that a variable named `_url` and a method named `_url` doesn't actually return a URL but instead returns the onclick handler.  Think this might be clearer to rename to `schedule_onclick` and the method to `report_action_onclick`